service/test/integration2_test.go

```diff
@@ -3156,6 +3156,11 @@ func TestBreakpointVariablesWithoutG(t *testing.T) {
 
 func TestGuessSubstitutePath(t *testing.T) {
 	t.Setenv("NOCERT", "1")
+	ver, _ := goversion.Parse(runtime.Version())
+	if ver.IsDevelBuild() && os.Getenv("CI") != "" && runtime.GOOS == "linux" {
+		t.Skip("does not work in TeamCity + tip + linux")
```

Comment:
Can you add a quick comment for reference on the TeamCity issue that's causing this failure.